airflow/providers/tableau/provider.yaml

```diff
@@ -37,6 +37,7 @@ integrations:
 operators:
   - integration-name: Tableau
     python-modules:
+      - airflow.providers.tableau.operators.tableau
```

Comment:
Would be nice also to add docs about the operator usage.
This needs to be added to:
https://github.com/apache/airflow/tree/main/docs/apache-airflow-providers-tableau

Examples from other operators:
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-amazon/operators/emr.rst
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-mysql/operators.rst
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-asana/operators/asana.rst

Reply:
Done